release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -538,6 +538,21 @@ Validate that SIG Cluster Lifecycle has all of the docs in place for the upcomin
 
 > Hi Sig Cluster Lifecylce :wave:  1.21 Docs Lead here, can someone confirm that all docs are in place for the upcoming 1.21 release?
 
+### Update Releases Page (the week before the release)
+
+Create a PR against the dev-[future-release] branch to add an entry for [future-release] on https://kubernetes.io/releases. 
+The file to update is https://github.com/kubernetes/website/blob/main/data/releases/schedule.yaml.
+The following fields are required:
+  - `release`: <1.xy>
+  - `releaseDate`: <YYYY-MM-DD> # date of release
+  - `next`: <1.xy.1>
+  - `cherryPickDeadline`: <YYYY-MM-DD> # the Friday before `targetDate`
```

Comment:
I don't think this cherry pick deadline date is actually needed prior to the release day, but adding it doesn't hurt.

Reply:
100% also, it saves from making a second PR later on.